types/react-image-magnify/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
This file should just be `{ "extends": "dtslint/dt.json" }`.

Reply:
Done